programs/Makefile

```diff
@@ -252,29 +254,29 @@ pkey/rsa_encrypt$(EXEXT): pkey/rsa_encrypt.c $(DEP)
 	echo "  CC    pkey/rsa_encrypt.c"
 	$(CC) $(LOCAL_CFLAGS) $(CFLAGS) pkey/rsa_encrypt.c    $(LOCAL_LDFLAGS) $(LDFLAGS) -o $@
 
-psa/aead_demo$(EXEXT): psa/aead_demo.c $(DEP)
-	echo "  CC    psa/aead_demo.c"
-	$(CC) $(LOCAL_CFLAGS) $(CFLAGS) psa/aead_demo.c    $(LOCAL_LDFLAGS) $(LDFLAGS) -o $@
+$(TF-PSA-CRYPTO_PROGRAM_PATH)/psa/aead_demo$(EXEXT): $(TF-PSA-CRYPTO_PROGRAM_PATH)/psa/aead_demo.c $(DEP)
```

Comment:
We should probably also update the path of the `psa/psa_constant_names_generated.c` target. Rather `$(TF-PSA-CRYPTO_PROGRAM_PATH)/psa/psa_constant_names_generated.c`.